From 428cb45c434bce7137ae47ced4e0b21113f095f0 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 11 Nov 2010 11:23:00 -0600
Subject: [PATCH] omap: devkit8000 fix regulator

The previous definition will cause following error message:
 twl_reg twl_reg.6: can't register VMMC1, -22
 twl_reg: probe of twl_reg.6 failed with error -22

because set_consumer_device_supply only allowed one of consumer_dev and
consumer_dev_name may be specified.

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 arch/arm/mach-omap2/board-devkit8000.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-omap2/board-devkit8000.c b/arch/arm/mach-omap2/board-devkit8000.c
index 51d2c80..2e14168 100644
--- a/arch/arm/mach-omap2/board-devkit8000.c
+++ b/arch/arm/mach-omap2/board-devkit8000.c
@@ -157,7 +157,7 @@ static void devkit8000_panel_disable_dvi(struct omap_dss_device *dssdev)
 }
 
 static struct regulator_consumer_supply devkit8000_vmmc1_supply =
-	REGULATOR_SUPPLY("vmmc", "mmci-omap-hs.0");
+	REGULATOR_SUPPLY("vmmc", NULL);
 
 
 /* ads7846 on SPI */
-- 
1.7.1

